toml/src/main/resources/META-INF/toml-only.xml

```diff
@@ -31,6 +31,10 @@
                          displayName="Crate not found"
                          enabledByDefault="true" level="WARNING"
                          implementationClass="org.rust.toml.inspections.CrateNotFoundInspection"/>
+        <localInspection language="TOML" groupName="Rust"
```

Comment:
Probably it makes sense here to have `Cargo.toml` dependency inspections in a separate group:
```suggestion
        <localInspection language="Rust" groupPath="Rust,Cargo.toml" groupName="Dependencies"
```

Reply:
Hmm, if I put `Rust` there, the inspection tests no longer invoke the crate not found/invalid inspections. Maybe it won't run Rust inspections in `.toml` files?

Reply:
Oh, sure 😅
```
<localInspection language="TOML" groupPath="Rust,Cargo.toml" groupName="Dependencies"
```

Reply:
Are we sure about `Rust/Cargo.toml/Dependencies` ? Maybe `Rust/Cargo.toml` is enough for now?
We don't have a lot of inspections for Cargo.toml and suggested structure maybe too complicated
![image](https://user-images.githubusercontent.com/2539310/114393450-b1819980-9ba2-11eb-96ce-213773623461.png)